src/debug.c

```diff
@@ -1201,25 +1201,25 @@ static void *getAndSetMcontextEip(ucontext_t *uc, void *eip) {
         return old_val;                         \
     } while (0)
 #if defined(__APPLE__) && !defined(MAC_OS_10_6_DETECTED)
-/* OSX < 10.6 */
+/* Mac OS X < 10.6 */
 #if defined(__x86_64__)
     GET_SET_RETURN(uc->uc_mcontext->__ss.__rip, eip);
 #elif defined(__i386__)
     GET_SET_RETURN(uc->uc_mcontext->__ss.__eip, eip);
 #else
-    /* OSX PowerPC */
+    /* Mac OS X PowerPC */
```

Comment:
PowerPC support was dropped (after 10.6) shortly before Apple rebranded (10.8).
https://en.wikipedia.org/wiki/MacOS_version_history

Reply:
I'm fairly confident it isn't.

I'm concurrently involved in a PyPI ticket that talks about that.

...All of this stems from the fact that I have a rule complaining about misspelling `macOS` in various ways. ...

The end of life for PPC was 2011 which is 14 years ago so it's pretty safe to argue that it's been obsolete for 10+ years.